sct.py

```diff
@@ -1105,5 +1105,15 @@ def clean_runner_instances():
     clean_sct_runners()
 
 
+@cli.command("remove-runner-on-success", help="Remove runner instance on test success")
+@click.argument("runner_ip", type=str)
+@click.argument("backend", type=str)
+@click.argument("region", type=str)
+def remove_runner_on_success(runner_ip: str,
+                             backend: str,
+                             region: str):
+    remove_test_runner_on_success(runner_ip=runner_ip, backend=backend, region=region)
```

Comment:
Need to add `add_file_logger()` here.

Reply:
Would that make sense if we're running this step after the "collect log data" stage?

Reply:
Yeah, it's more about writing logs to the files than collecting them to store in S3.

Reply:
@KnifeyMoloko we need to the logger to save the command output to file in order to debug when we have issues

Reply:
Makes sense. This is currently outdated, since I moved to using the `clean_runner_instances()` function rather than a separate new function